clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1b_machine_set.yaml

```diff
@@ -29,14 +29,15 @@ spec:
           apiVersion: awsproviderconfig.openshift.io/v1beta1
           blockDevices:
           - ebs:
-              volumeSize: 1500
-              volumeType: gp2
+              volumeSize: 500
```

Comment:
In the worst case scenario of us needing to delete the job specific machine sets (or if something goes wrong with the webhook), these nodes will take the load. 500G (since tests need significant storage) and 5000 IOPS (since builds need significant performance) prepare for a worst case scenario where workers are back to handling arbitrary workloads. 
In the past, it was set to 1500 gp2 to artificially increase IOPS. With gp3, this isn't necessary.

Reply:
```console
oc --context build01 get machineset -n openshift-machine-api  build01-9hdwj-ci-builds-worker-us-east-1a -o yaml | grep -A6 blockDevices:
          blockDevices:
          - ebs:
              iops: 5000
              kmsKey: {}
              throughput: 500
              volumeSize: 200
              volumeType: gp3
```
You have tested them already.  :)